db/schema.rb

```diff
@@ -22,6 +22,18 @@
     t.datetime "updated_at", null: false
     t.text "description"
     t.string "image"
+    t.string "of_type"
+    t.boolean "status"
+  end
+
+  create_table "restaurants", force: :cascade do |t|
+    t.string "name"
+    t.string "address"
+    t.integer "phonenumber"
+    t.string "hours"
```

Comment:
Do we need an "order-hours" or something like that? :) Since it will be different from the opening hours. 9.30-18.30. Just something to consider. We need to be able to turn off the ordering system outside of those hours, according to the client.

Reply:
Good idea

Reply:
hours is not a very descriptive name for the  column